hardware/EvohomeBase.h

```diff
@@ -740,8 +740,9 @@ class CEvohomeBase : public CDomoticzHardwareBase
 		cmEvoAutoWithEco, //  0x01
 		cmEvoAway,	  //  0x02
 		cmEvoDayOff,	  //  0x03
-		cmEvoCustom,	  //  0x04
-		cmEvoHeatingOff,  //  0x05
+                cmEvoDayOffWithEco,      //  0x04
```

Comment:
tab vs space?

Reply:
Tabs please

Reply:
I have literally copied and pasted lines from above. So I never bother checking if they have tabs or spaces for the indents.

Reply:
@brucemiranda , no worries, but I don't known what development environment you use, if you use Visual Studio it will correct it, or when doing control-k followed by control-d

Reply:
Good ol' nano on Ubuntu. To try and stay consistent, I always just copy and paste.

Reply:
Yes best to use a IDE. If you don't use windows I would advise to use kdevelop

Reply:
Don't see myself as a developer, so never thought I would need the tools needed :-)

Reply:
Okey, still, please use a TAB here... you can continue committing and pushing and this PR will update

Reply:
Hopefully that is now all done and this PR can be merged?